renderer/src/components/ManualInputForm/ManualInputForm.tsx

```diff
@@ -48,7 +48,13 @@ export default function ManualInputForm({
 
   useEffect(() => {
     setShowDeleteMessage(false);
+  }, [selectedDate]);
+
+  useEffect(() => {
+    setReportHandler(selectedDateReport);
+  }, [selectedDateReport]);
 
+  useEffect(() => {
     (async () => {
```

Comment:
Move the logic inside the useEffect hook into a new function and invoke it here, please.

Reply:
It'd be nice to name this function with a sense and move it into another file.